### PR TITLE
Update schedule payment

### PR DESCRIPTION
**Pull request by dara123**

### CODE REVIEW

File: src/Services/PaymentService.php [hunk 1]
```diff
@@ -124,4 +124,19 @@ public function cancel($paymentId)
...
+     * @param array|JsonSerializable|Entities\Payment $payment
+     * @param string $paymentId
+     *
+     * @throws NotFoundException The payment does not exist
+     * @throws NotFoundException The scheduled payment does not exist
+     * @throws GoneException The process is completed but the payment can not be created
+     *
+     * @return Entities\ScheduledPayment|Entities\Payment
+     */
+    public function update($payment, $paymentId)
```
Comment (slavcodev) [inline src/Services/PaymentService.php]: To keep consistency, we need to flip arguments. The required URL params have more weight then request body.

Comment (dara123) [inline src/Services/PaymentService.php]: @slavcodev 
``` public function create($payment, $paymentId) ```
should we keep the same order as create?

**Comment by adamaltman** [comment]
Maybe this should be in a new class, `ScheduledPaymentService`?

**Comment by adamaltman** [comment]
Please see #138 ,  I think we can close this PR and use #138 .

**Comment by dara123** [comment]
:+1:

**Pull request closed without merging**